Descent3/d3movie.cpp

```diff
@@ -357,17 +320,7 @@ void CallbackShowFrameNoFlip(unsigned char *buf, unsigned int bufw, unsigned int
 intptr_t mve_SequenceStart(const char *mvename, void *fhandle, oeApplication *app, bool looping) {
 #ifndef NO_MOVIES
   // first, find that movie..
-  std::filesystem::path real_name;
-#if defined(POSIX)
-  real_name = mve_FindMovieFileRealName(mvename);
-  if (real_name.empty()) {
-    LOG_WARNING.printf("MOVIE: No such file %s", mvename);
-    fhandle = nullptr;
-    return 0;
-  }
-#else
-  real_name = mvename;
-#endif
+  std::filesystem::path real_name = cf_LocatePath(std::filesystem::path("movies") / mvename);
```

Comment:
Probably `mvename` should be inside `cf_LocatePath()`? It can be non-lowercased.

Reply:
I’m confused. To me, it looks like `mvename` is inside of the `cf_LocatePath()` call. Could you post a diff of what you’re suggesting here?